### ISSUE TITLE
Ballerina to swagger converter tool doesn't add operation parameter schema correctly

### ISSUE THREAD

**Issue by NipunaMarcus** [ballerina-platform/ballerina-lang#12550]
**Description:**
When ballerina file (.bal) which is to be converted to swagger has an operation with parameters and parameters has a type when converting this to swagger will not add the parameter type details to swagger.

**Comment by NipunaMarcus** [comment]
Closing this issue since we have revamped the OpenAPI tool and attached it to the new issue to track and evaluate once the new implementation is done.
